### ISSUE TITLE
Shadow Warrior Moral (+Other ranged moral 2)

### ISSUE THREAD

**Issue by Green-Wolf** [WarEmu/WarBugs#1067]
Just wanted to report a bug on shadow warrior.

On live when you fire an arrow (Like Eagle Eye) and then press your moral 2 (+100% damage) buff before the shot lands the double damage is applied to the arrow in the air. This was important because it let you precast long spells and not waste your 7 sec duration. It also let you get the shot off before you poped the moral, as currently sometimes you pop it and then get CCed during your cast  :evil: 

However currently this doesn't work in RoR.

Source - https://www.youtube.com/watch?v=nBV6gFYuqts - 0:49

**Comment by ozarmidrashim** [comment]
Actually, it does work. Damage calculation is done when the arrow hits the target and not before.

**Comment by Green-Wolf** [comment]
Really? I shall go recheck, my apologies if so, I must be doing something wrong!

**Comment by Green-Wolf** [comment]
Just tried to do it again and again for 5 scenarios, it doesn't work for me. I press 1 for eagle eye, as soon as it goes off I hit 4 for moral 2 and the damage is the same as with no moral. Perhaps someone else can test it.

**Comment by xyeppp** [comment]
Works as intended on SH with same morale.

**Comment by Green-Wolf** [comment]
I just logged on and went into a scenario and it was working, then I realised that because I had just logged on my tactics had wiped. 

I put my tactics back in (No Quarter & Masterful Aim) and tried it again, now it was broken again. Perhaps it has something to do with 'No Quarter', which reduces the cast time of eagle eye from 2 sec to 1 sec. 

Could anyone else check on their own shadow warrior using those tactics?
